modules/items/tests/client/admin.items.client.controller.tests.js

```diff
@@ -1,183 +0,0 @@
-      var sampleItemPostData;
-
-      beforeEach(function () {
-        // Create a sample item object
-        sampleItemPostData = new ItemsService({
-          title: 'An Item about MEAN',
-          content: 'MEAN rocks!'
-        });
-
-        $scope.vm.item = sampleItemPostData;
-      });
-
-      it('should send a POST request with the form input values and then locate to new object URL', inject(function (ItemsService) {
-        // Set POST response
-        $httpBackend.expectPOST('/api/items', sampleItemPostData).respond(mockItem);
-
-        // Run controller functionality
-        $scope.vm.save(true);
-        $httpBackend.flush();
-
-        // Test Notification success was called
-        expect(Notification.success).toHaveBeenCalledWith({ message: '<i class="glyphicon glyphicon-ok"></i> Item saved successfully!' });
-        // Test URL redirection after the item was created
-        expect($state.go).toHaveBeenCalledWith('admin.items.list');
-      }));
-
-      it('should call Notification.error if error', function () {
-        var errorMessage = 'this is an error message';
-        $httpBackend.expectPOST('/api/items', sampleItemPostData).respond(400, {
-          message: errorMessage
-        });
-
-        $scope.vm.save(true);
-        $httpBackend.flush();
-
-        expect(Notification.error).toHaveBeenCalledWith({ message: errorMessage, title: '<i class="glyphicon glyphicon-remove"></i> Item save error!' });
-      });
-    });
-
-    describe('vm.save() as update', function () {
-      beforeEach(function () {
-        // Mock item in $scope
-        $scope.vm.item = mockItem;
-      });
-
-      it('should update a valid item', inject(function (ItemsService) {
-        // Set PUT response
-        $httpBackend.expectPUT(/api\/items\/([0-9a-fA-F]{24})$/).respond();
-
-        // Run controller functionality
-        $scope.vm.save(true);
-        $httpBackend.flush();
-
-        // Test Notification success was called
-        expect(Notification.success).toHaveBeenCalledWith({ message: '<i class="glyphicon glyphicon-ok"></i> Item saved successfully!' });
-        // Test URL location to new object
-        expect($state.go).toHaveBeenCalledWith('admin.items.list');
-      }));
-
-      it('should  call Notification.error if error', inject(function (ItemsService) {
-        var errorMessage = 'error';
-        $httpBackend.expectPUT(/api\/items\/([0-9a-fA-F]{24})$/).respond(400, {
-          message: errorMessage
-        });
-
-        $scope.vm.save(true);
-        $httpBackend.flush();
-
-        expect(Notification.error).toHaveBeenCalledWith({ message: errorMessage, title: '<i class="glyphicon glyphicon-remove"></i> Item save error!' });
-      }));
-    });
-
-    describe('vm.remove()', function () {
-      beforeEach(function () {
-        // Setup items
-        $scope.vm.item = mockItem;
-      });
-
-      it('should delete the item and redirect to items', function () {
```

Comment:
I know we are making items accessible to all users instead of just admins, but these tests can be converted. They are super useful still

Reply:
I think this is a artefact from where I branched off of. I didn't intend to change any test files on this PR.

Reply:
I think this is a artefact from where I branched off of. I didn't intend to change any test files on this PR.

Reply:
The only filechanges should be directly related to incrementing. Not sure why so many have updates but I need to fix that.